grunt/config/jscs.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+    src: [
+        'src/**/*.js',
+        '!src/core/js/libraries/*.js'
+    ],
+    options: {
+        reporter: "unix",
+        fix: true,
+        // rules
+        disallowMixedSpacesAndTabs: true,
+        disallowMultipleVarDecl: "exceptUndefined",
+        disallowNewlineBeforeBlockStatements: true,
+        disallowQuotedKeysInObjects: true,
+        disallowSpaceAfterObjectKeys: true,
+        disallowSpaceAfterPrefixUnaryOperators: true,
+        disallowSpacesInFunction: {
+            beforeOpeningRoundBrace: true
+        },
+        disallowTrailingWhitespace: true,
+        disallowTrailingComma: true,
+        requireCamelCaseOrUpperCaseIdentifiers: true,
+        requireCapitalizedConstructors: true,
+        requireCurlyBraces: true,
```

Comment:
I think for `requireCurlyBraces` we'd need a few exceptions to prevent existing code from throwing errors and to allow for clean code e.g. 
` if (parameterNotSet) return;`
at the start of a function

Also for `case` and `default` in `switch` statements - I don't think we ever use curly braces with these